source/agora/node/Ledger.d

```diff
@@ -1352,3 +1353,88 @@ unittest
     genBlockTransactionsFreeze(1, TxType.Payment);
     assert(ledger.getBlockHeight() == 11 + 2016);
 }
+
+/// test enrollments in the genesis block
+unittest
+{
+    import agora.common.BitField;
+    import std.exception;
+
+    class PreLoadedMemBlockStorage : MemBlockStorage
```

Comment:
```suggestion
    static class PreLoadedMemBlockStorage : MemBlockStorage
```

Nitpick

Reply:
Ha I fixed that too before reading your review.